src/Chapter1.hs

```diff
@@ -652,9 +676,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n 
+  | n < 100 = n `div` 10
+  | n < 1000 = n `div` 100
+  | n < 10000 = n `div` 1000
+  | n < 100000 = n `div` 10000
```

Comment:
I'm afraid, this solution doesn't work for all the numbers 😞 
You need to implement a recursive function that will divide the number by 10 unless only a single digit left